### PR TITLE
lint `graph/_*.py`

### PR DESCRIPTION
**Pull request by jGaboardi**
xref https://github.com/pysal/libpysal/issues/589

This PR:
* formats and lints `libpysal/graph/_*.py`
* rename `_set_ops._Set_Mixin` to `_set_ops.SetOpsMixin`

### CODE REVIEW

**Comment by martinfleis** [comment]
> I can add the remaining 2 files (_{contiguity, _kernel}.py) in here

include those.

> What do we think about renaming _set_ops._Set_Mixin --> _set_ops._SetMixin? Or even something like _set_ops .SetOpsMixin to conform with naming conventions? 

Whatever, this is fully private. Happy to stick to strict CamelCase if it lives in the private module anyway

**Comment by knaaptime** [comment]
swap it james. I dont like ignoring pep8 anyway. i'll be psyched once we finally get rid of those underscores in esda

File: libpysal/graph/_kernel.py [hunk 1]
```diff
@@ -300,7 +306,7 @@ def _prepare_tree_query(coordinates, metric, p=2):
         return tree(coordinates, metric=metric, **dist_kwds).query
     else:
         if metric in ("euclidean", "manhattan", "cityblock", "minkowski"):
-            from scipy.spatial import KDTree as tree
+            from scipy.spatial import KDTree as tree  # noqa N813
 
             tree_ = tree(coordinates)
```
Comment (martinfleis) [inline libpysal/graph/_kernel.py:311]:
```suggestion
            tree_ = spatial.KDTree(coordinates)
```

File: libpysal/graph/_parquet.py [hunk 2]
```diff
@@ -1,8 +1,9 @@
-import libpysal
 import json
 
+import libpysal
+
 
-def _to_parquet(G, destination, **kwargs):
+def _to_parquet(G, destination, **kwargs):  # noqa N803
```
Comment (martinfleis) [inline libpysal/graph/_parquet.py:6]: We can use something else than `G` here, to comply.

File: libpysal/graph/_plotting.py [hunk 3]
```diff
@@ -4,7 +4,7 @@
 
 
 def _plot(
-    G,
+    G,  # noqa N803
```
Comment (martinfleis) [inline libpysal/graph/_plotting.py:7]: As above.

File: libpysal/graph/_utils.py [hunk 4]
```diff
@@ -137,7 +137,7 @@ def _build_coincidence_lookup(geoms):
     """
     Identify coincident points and create a look-up table for the coincident geometries.
     """
-    valid_coincident_geom_types = set(("Point",))
+    valid_coincident_geom_types = set(("Point",))  # noqa C405
```
Comment (martinfleis) [inline libpysal/graph/_utils.py:140]: I remember some issue with this but why do we add `noqa` instead of `{("Point",)}`? The latter does not work?

File: libpysal/graph/_utils.py [hunk 5]
```diff
@@ -224,7 +224,7 @@ def _validate_sparse_input(sparse, ids=None):
     return _sparse_to_arrays(sparse, ids)
 
 
-def _vec_euclidean_distances(X, Y):
+def _vec_euclidean_distances(X, Y):  # noqa N803
```
Comment (martinfleis) [inline libpysal/graph/_utils.py:227]: Can we just use lowercase x and y?

**Review by martinfleis — COMMENTED** [review]
Two more notes, sorry

File: libpysal/graph/_utils.py (hunk 4 above)
Comment (jGaboardi) [reply libpysal/graph/_utils.py:140]: I tried `{("Point",)}` and it broke CI. But let me try again to double check.

File: libpysal/graph/_utils.py (hunk 5 above)
Comment (jGaboardi) [reply libpysal/graph/_utils.py:227]: I'll go with `x_vec, y_vec`. I think easier to read as the difference between scalar and vector.

File: libpysal/graph/_utils.py (hunk 4 above)
Comment (jGaboardi) [reply libpysal/graph/_utils.py:140]: OK, se we get a ton of these: `ValueError: coindicence checks are only well-defined for geom_types: {('Point',)}`

For example:
```
libpysal/graph/tests/test_set_ops.py::Test_Set_Ops::test___len__ - ValueError: coindicence checks are only well-defined for geom_types: {('Point',)}
```

And then we also get several: `AssertionError: Regex pattern did not match.`

For example:
```
libpysal/graph/tests/test_triangulation.py::Test_Coincident::test_voronoi_error - AssertionError: Regex pattern did not match.
```

Comment (jGaboardi) [reply libpysal/graph/_utils.py:140]: Where the `valid_geometry_types` come from [`_VALID_GEOMETRY_TYPES`](https://github.com/pysal/libpysal/blob/ed98cbd797beaa03fcb479e2f4fa915266b99281/libpysal/graph/_triangulation.py#L32)

Comment (martinfleis) [reply libpysal/graph/_utils.py:140]: Okay, leave it.

**Review by martinfleis — APPROVED** [review]
